patterns/atoms/buttons/primary.php

```diff
@@ -1,2 +1,17 @@
-<p><a href="#" class="a__btn a__btn__primary">Click Me!</a></p>
-<p><a href="#" class="a__btn a__btn__primary" disabled="disabled">Disabled</a></p>
+<?php
```

Comment:
Yes I agree, I just thought the code was there for any reason, so I just added the parameters (didn't want to break the harmony! haha)

I'll make the changes, thanks!